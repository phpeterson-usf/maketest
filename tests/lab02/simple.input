$(TESTS_DIR)/simple.ntc
