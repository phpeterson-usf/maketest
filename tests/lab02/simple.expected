TK_INTLIT("1")
TK_PLUS("+")
TK_INTLIT("2")
TK_EOL("")
TK_EOT("")
